### ISSUE TITLE
Error when updating codepen

### ISSUE THREAD

**Issue by aturkewi** [learn-co-curriculum/selecting-single-elements-lab#12]
Student was trying to complete the code along as per the lesson instructions but was receiving an error in the console whenever he would updated the CodePen editor with new code. See attached screenshot - it was throwing this error instead of logging to the console.
I made sure he had the console set to the CodePen example and the logging to verbose. Refreshing the page didn't help.
Another user (email removed) also said she was having the same issue.

![codepen-editor](https://user-images.githubusercontent.com/13033515/27290858-7e45a770-54dc-11e7-8fde-d3c6a17e1021.png)

Originally posted from: https://flatiron.atlassian.net/browse/TS-373 by @cassaram09

**Comment by aturkewi** [comment]
@jmburges

**Comment by gj** [comment]
Hey @aturkewi !

I'm not sure how to follow-up with the students (neither seem to have a connected GitHub account), but our best guess (credit to @jmburges) is that the student typed the bare-word `Copy` into the CodePen JS tab, the error(s) appeared in the Chrome JS console, and then the student erased `Copy` but the errors persisted (because the Chrome JS console hadn't been cleared).

I've added a little note to the lesson about how/why to clear the Chrome JS console. Let me know if other students start running into similar issues.

I've also opened #13 as a general reminder to add a note somewhere early in the JS curriculum about common types of JS console errors (like `ReferenceError`s) that students might encounter.
